src/Microsoft.AspNet.Mvc.ApiExplorer/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "description": "The API explorer component for ASP.NET MVC.",
```

Comment:
Can we say a tiny bit more here? Like what it does? (Not necessarily a very long description...)